package.json

```diff
@@ -30,6 +30,7 @@
     "clean:doc:kss": "rm -rf build/styleguide",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "deploy:doc": "git-directory-deploy --directory build/ --branch gh-pages",
+    "icon": "scripts/generate-svgr-icon.sh",
```

Comment:
generate-svgr-icon seems more explicit, I prefer to have verbs for things in package.json scripts.

Reply:
@ptbrowne est-ce que "makeSvgr" t'irait ? histoire de faire plus court étant donné qu'il n'y a pas d'autocompletion